zanata-war/src/main/webapp/WEB-INF/faces-config.xml

```diff
@@ -118,36 +118,6 @@
     </navigation-case>
   </navigation-rule>
 
-  <!-- Landing page for Kerberos ticket-based Authentication -->
-  <navigation-rule>
-    <from-view-id>/account/klogin.xhtml</from-view-id>
-    <navigation-case>
-      <if>#{authenticationManager.authenticationRedirect eq 'login'}</if>
-      <to-view-id>/account/login.xhtml</to-view-id>
-      <redirect />
-    </navigation-case>
-    <navigation-case>
-      <if>#{authenticationManager.authenticationRedirect eq 'edit'}</if>
-      <to-view-id>/profile/create_user.xhtml</to-view-id>
-      <redirect />
-    </navigation-case>
-    <navigation-case>
-      <if>#{authenticationManager.authenticationRedirect eq 'inactive'}</if>
-      <to-view-id>/account/inactive_account.xhtml</to-view-id>
-      <redirect />
-    </navigation-case>
-    <navigation-case>
-      <if>#{authenticationManager.authenticationRedirect eq 'dashboard'}</if>
-      <to-view-id>/dashboard/home.xhtml</to-view-id>
-      <redirect />
-    </navigation-case>
-    <navigation-case>
-      <if>#{authenticationManager.authenticationRedirect eq 'home'}</if>
```

Comment:
Perhaps it's a shame we can't control this in faces-config.xml any more, but on the other hand, this was a bit ugly...